project.xml

```diff
@@ -73,46 +73,63 @@
         </use>
     </use>
 
-    <use project = "fty-common-mlm" libname = "libfty_common_mlm" header = "fty_common_mlm.h"
-        repository = "https://github.com/42ity/fty-common-mlm.git"
-        release = "master"
-        test = "fty_common_mlm_selftest" >
+    <use project = "fty-common-nut" libname = "libfty_common_nut" header = "fty_common_nut.h"
+            repository = "https://github.com/42ity/fty-common-nut.git"
+            release = "master"
+            test = "fty_common_nut_selftest" >
 
-        <use project = "czmq"
-            repository = "https://github.com/42ity/czmq.git"
-            release = "v3.0.2-FTY-master"
-            min_major = "3" min_minor = "0" min_patch = "2" >
+        <use project = "fty-common-mlm" libname = "libfty_common_mlm" header = "fty_common_mlm.h"
```

Comment:
this block should be after '<use project = "fty-proto"'
@geraldguillaume @jimklimov @perrettecl we should definitely have a task in Jim's backlog to address automatic zproject dependencies (i.e. declare the "leaf" projects, which dependencies will be pulled) to avoid the too-many duplicates and ordering issues we have again and again. A short run implem could be to fork zproject and modify zproject_known_projects.xml with our common projects and deps